src/libraries/drand/_index.md

```diff
@@ -0,0 +1,120 @@
+
+After initializing access to a drand beacon, a Filecoin node should have access to the following values:
+
+- `filEpochDuration`    -- the Filecoin network's epoch duration (between any two leader elections)
+- `filGenesisTime`      -- the Filecoin genesis timestamp
+- `filEpoch`            -- the current Filecoin epoch
+- `drandGenesisTime`    -- drand's genesis timestamp
+- `drandPeriod`         -- drand's epoch duration (between any two randomness creations)
+
+Using the above, a Filecoin node can determine the appropriate drand round value to be used for use in {{<sref leader_election>}} in an epoch using both networks' reliance on real time as follows:
+
+```go
+MaxBeaconRoundForEpoch(filEpoch) {
+    // determine what the latest Filecoin timestamp was from the current epoch number
+    var latestTs
+    if filEpoch == 0 {
+        latestTs = filGenesisTime
+    } else {
+        latestTs = ((uint64(filEpoch) * filEpochDuration) + filGenesisTime) - filEpochDuration
+    }
+    // determine the drand round number corresponding to this timestamp
+    dround := (latestTs - drandGenesisTime) / uint64(drandPeriod)
+    return dround
+}
+```
+
+### Edge cases and dealing with a drand outage
+
+It is important to note that any drand beacon outage will effectively halt Filecoin block production. Given that new randomness is not produced, Filecoin miners cannot generate new blocks.
+
+After a beacon downtime, drand nodes will work to quickly catch up to the current round, as defined by wall clock time. In this way, the above time-to-round mapping in drand (see above) used by Filecoin remains an invariant after this catch-up following downtime.
+
+So while Filecoin miners were not able to mine during the drand outage, they will quickly be able to run leader election thereafter, given a rapid production of drand values. We call this a "catch up" period.
+
+During the catch up period, Filecoin nodes will backdate their blocks in order to continue using the same time-to-round mapping to determine which drand round should be integrated according to the time. Miners can then choose to publish their null blocks for the outage period (including the appropriate drand entries throughout the blocks, per the time-to-round mapping), or (as is more likely) try to craft valid blocks that might have been created during the outage.
+
+Note that based on the level of decentralization of the Filecoin network, we expect to see varying levels of miner collaboration during this period. This is because there are two incentives at play: trying to mine valid blocks from during the outage to collect block rewards, not falling behind a heavier chain being mined by a majority of miners that may or may not have ignored a portion of these blocks.
+
+In any event, a heavier chain will emerge after the catch up period and mining can resume as normal.
+
+### Filecoin's drand network characteristics
```

Comment:
I would not call that `Filecoin's drand network` but `drand network specification` ;) 
And yes, once the main network is launched we can give info on how to connect - note the network is meant to be evolving so I wouldn't stick too much of what is the composition of the network nor the governance in a spec that is supposed to be fixed.